Core/src/main/java/com/plotsquared/core/location/UncheckedWorldLocation.java

```diff
@@ -60,6 +60,19 @@ private UncheckedWorldLocation(
         return new UncheckedWorldLocation(world, x, y, z);
     }
 
+    /**
+     * Construct a new location with yaw and pitch equal to 0
+     *
+     * @param world World
+     * @param loc   Coordinates
+     * @return New location
+     * @since 6.9.0
```

Comment:
This is likely wrong?

Reply:
Ah yeah that'll be because I copy-pasted it